docker/docker-compose.1804.53.yaml

```diff
@@ -6,7 +6,7 @@ services:
     image: swift-nio-http2:18.04-5.3
     build:
       args:
-        base_image: "swiftlang/swift:nightly-master-bionic"
```

Comment:
I presume this change was not supposed to come along?

Reply:
Heh. Some confusion on my part it seems around being behind master. I will remove it